dvc/remote/hdfs.py

```diff
@@ -65,6 +94,7 @@ def _group(regex, s, gname):
         return match.group(gname)
 
     def get_file_checksum(self, path_info):
+        # NOTE: pyarrow doesn't support checksum, so we need to use hadoop
```

Comment:
Depending on how this is implemented it might be faster to just open a file and calculate checksum yourself. This will also remove dep on hadoop cli.

Reply:
@Suor I agree, but I was not able to confirm that yet. We've discussed it during our private call :slightly_smiling_face: For now, tests went from 20minutes to 12minutes on cron jobs on py3.7, which is very good. Dropping hadoop cli there would make it even better, I will be sure to take a look at it.

Reply:
I also see `.info()` method in pyarrow, does it have checksum there?

Reply:
It doesn't.

Reply:
Out of luck) we should look into adding checksum to pyarrow then, or at least open an issue.